bears/general/KeywordBear.py

```diff
@@ -34,20 +55,57 @@ def run(self,
             for keyword in keywords_case_sensitive:
                 results += self.check_line_for_keyword(line,
                                                        filename,
+                                                       file,
                                                        line_number,
-                                                       keyword)
+                                                       keyword,
+                                                       dependency_results)
 
             for keyword in keywords_case_insensitive:
                 results += self.check_line_for_keyword(line.lower(),
                                                        filename,
+                                                       file,
                                                        line_number,
-                                                       keyword)
+                                                       keyword,
+                                                       dependency_results)
 
         return results
 
-    def check_line_for_keyword(self, line, filename, line_number, keyword):
+    def check_line_for_keyword(
+            self,
+            line,
+            filename,
+            file,
+            line_number,
+            keyword,
+            dependency_results):
+
         pos = line.find(keyword)
+        comments = _get_comments(dependency_results)
         if pos != -1:
+            diffs = None
+            for source_range_comment in comments:
+                todo_source_range = SourceRange.from_values(
+                    filename, line_number+1, pos+1)
+                if todo_source_range not in source_range_comment:
+                    continue
+
+                diff = Diff(file)
+                # remove just part of the line
+                comment_start = source_range_comment.start.column
+                line_before_todo_comment = line[:comment_start-1].strip()
+                if (line_before_todo_comment and
+                        line_number + 1 == source_range_comment.start.line):
+                    diff.change_line(
+                        line_number + 1,
+                        line,
+                        line.replace(line[comment_start - 1:],
+                                     '').rstrip() + '\n'
+                    )
```

Comment:
I'm not sure what standard we follow at coala, but from what I've seen, this bracket goes at the end of the last argument (after the `'\n'` in Line 102). Not sure, others can pip in here.